back/engines/commercial/analytics/app/models/analytics/fact_participation.rb

```diff
@@ -6,19 +6,19 @@
 #
 #  id                        :uuid             primary key
 #  dimension_user_id         :uuid
+#  participant_id            :uuid
 #  dimension_project_id      :uuid
-#  dimension_type_id         :uuid
 #  dimension_date_created_id :date
-#  reactions_count           :integer
-#  likes_count               :integer
-#  dislikes_count            :integer
+#  item_type                 :string
+#  action_type               :string
+#  reactable_type            :text
+#  reaction_id               :text
 #
 module Analytics
   class FactParticipation < Analytics::ApplicationRecordView
     self.primary_key = :id
     belongs_to :dimension_user, class_name: 'Analytics::DimensionUser'
     has_many :dimension_user_custom_field_values, class_name: 'Analytics::DimensionUserCustomFieldValue', foreign_key: :dimension_user_id, primary_key: :dimension_user_id
-    belongs_to :dimension_type, class_name: 'Analytics::DimensionType'
```

Comment:
I removed this relation because I was too lazy to write a complex join, and because this dimension was not strictly necessary for the current queries we do. But let me know if we should still add it for consistency